packages/gatsby/src/commands/develop.ts

```diff
@@ -71,12 +71,12 @@ const doesConfigChangeRequireRestart = (
 const getDebugPort = (port?: number): number => port ?? 9229
 
 export const getDebugInfo = (program: IProgram): IDebugInfo | null => {
-  if (program.hasOwnProperty(`inspect`)) {
+  if (Object.prototype.hasOwnProperty.call(program, `inspect`)) {
```

Comment:
I was fixing this rule but eventually moved to warning.

Reply:
Can you explain why this is needed?

Reply:
https://eslint.org/docs/rules/no-prototype-builtins

It's a security risk as technically program could have overwritten hasOwnProperty

Reply:
Neat. I'd not thought of that